packages/grid/_modules_/grid/hooks/features/rows/useGridRows.ts

```diff
@@ -120,9 +124,13 @@ export const useGridRows = (
     [apiRef],
   );
 
-  const loadRows = React.useCallback(
-    (startIndex: number, pageSize: number, newRows: GridRowModel[], rowCount?: number) => {
-      logger.debug(`Loading rows from index:${startIndex} to index:${startIndex + pageSize}`);
+  const insertRows = React.useCallback(
+    ({ startIndex, newRows, rowCount }: GridInsertRowParams) => {
+      logger.debug(
+        `Loading rows from index:${startIndex} to index:${
```

Comment:
It doesn't seem to describe the function.

Reply:
Updated